### ISSUE TITLE
Ability to save and continue working..

### ISSUE THREAD

**Issue by sokary2004** [Coding-Enthusiast/FinderOuter#66]
Can you add the possibility to continue working? Or the stop button... is it possible to add gpu too?

Thank you for creating such a beautiful program

**Comment by sokary2004** [comment]
1:Stop button - continue button - the program has the ability to stop and save and continue processing
2:Working with a graphics card increases the speed very much. The possibility of using multiple graphics cards and CPUs at the same time
3:The output of the program is automatically saved in a text file
4:The program can open a text file that has an incomplete private key and process it in order. in such a way that we leave the possible results in the desired file and do not need to enter them one by one into the program
5:Ability to select CPU manually (cpu thread)
6:Speed ​​display
7:Ability to recognize similar addresses (vanity) This makes us recognize the path better
8:Generate random content for each house or in general
9:Add the possibility of server and client. Cracking private keys with each other or with multiple servers
10: ......

**Comment by sokary2004** [comment]
app convert address to hash rmd160  and check vanity same time ..it is cool

**Comment by Coding-Enthusiast** [comment]
Pause/Resume capabilities have been on my list (#12) but since it requires a significant changes to the code I've postponed it to after releasing version 1.0 (ie after getting out of beta). It is also listed in the roadmap (#47).  
GPU support is also in the list (#47) for the next major release (2.0).  

> 6:Speed ​​display

You mean something like live report of key/sec speed? Isn't the progress bar enough?  

> 4:The program can open a text file that has an incomplete private key and process it in order. in such a way that we leave the possible results in the desired file and do not need to enter them one by one into the program
> 7:Ability to recognize similar addresses (vanity) This makes us recognize the path better
> 8:Generate random content for each house or in general
> 9:Add the possibility of server and client. Cracking private keys with each other or with multiple servers

FinderOuter is a recovery tool to be used by those who have lost access to their _own_ coins. The quoted stuff sounds like puzzle solving or those silly things where you search the key space to "get lucky".

**Comment by nerblock** [comment]
Can we at least get a stop button without resume functionality?

It would be extremely useful for testing and tweaking the settings, so one doesn't have to close/reopen the app and copypaste all the values again in the input fields.

**Comment by Humble2020** [comment]
> 1:Stop button - continue button - the program has the ability to stop and save and continue processing 2:Working with a graphics card increases the speed very much. The possibility of using multiple graphics cards and CPUs at the same time 3:The output of the program is automatically saved in a text file 4:The program can open a text file that has an incomplete private key and process it in order. in such a way that we leave the possible results in the desired file and do not need to enter them one by one into the program 5:Ability to select CPU manually (cpu thread) 6:Speed ​​display 7:Ability to recognize similar addresses (vanity) This makes us recognize the path better 8:Generate random content for each house or in general 9:Add the possibility of server and client. Cracking private keys with each other or with multiple servers 10: ......

The tool runs in parallel multi-threading already.. yeah its slow
